tests/src/Functional/ContentEventRenderTest.php

```diff
@@ -741,6 +766,76 @@ public function testEventRendering(): void {
     $timeline_html = $timeline_content->getOuterHtml();
     $timeline_assert->assertPattern($timeline_expected_values, $timeline_html);
+
+    // Create jobs for person entity.
+    $person_job_1 = PersonJob::create([
+      'type' => 'oe_default',
+      'oe_role_reference' => 'http://publications.europa.eu/resource/authority/role-qualifier/ADVIS',
+    ]);
+    $person_job_1->save();
+    $person_job_2 = PersonJob::create([
+      'type' => 'oe_default',
+      'oe_role_reference' => 'http://publications.europa.eu/resource/authority/role-qualifier/ADVIS_CHIEF',
+    ]);
+    $person_job_2->save();
+
+    /** @var \Drupal\node\Entity\Node $person */
+    $values = [
+      'type' => 'oe_person',
+      'oe_person_first_name' => 'Mick',
+      'oe_person_last_name' => 'Jagger',
+      'oe_content_owner' => 'http://publications.europa.eu/resource/authority/corporate-body/ABEC',
+      'uid' => 0,
+      'status' => 1,
+    ];
+    $person = Node::create($values);
+    $person->set('oe_person_jobs', [$person_job_1, $person_job_2]);
+    $person->save();
+
+    $event_speaker = EventSpeaker::create([
+      'oe_event_role' => 'event role 1',
+      'type' => 'oe_default',
+      'oe_person' => $person,
+    ]);
+    $event_speaker->save();
+
+    $node->set('oe_event_speakers', [$event_speaker]);
+    $node->save();
+
+    $this->drupalGet($node->toUrl());
+    $speakers = $this->assertSession()->elementExists('css', '.ecl-row.field-oe-event-speakers');
+    $speakers_items = $speakers->findAll('css', '.ecl-u-d-flex.ecl-u-pv-m.ecl-u-border-bottom.ecl-u-border-color-grey-15.ecl-col-12.ecl-col-m-6.ecl-col-l-4');
+    $this->assertCount(1, $speakers_items);
+    $node->set('oe_event_speakers', [$event_speaker, $event_speaker]);
+    $node->save();
+
+    $this->drupalGet($node->toUrl());
+    $speakers = $this->assertSession()->elementExists('css', '.ecl-row.field-oe-event-speakers');
+    $speakers_items = $speakers->findAll('css', '.ecl-u-d-flex.ecl-u-pv-m.ecl-u-border-bottom.ecl-u-border-color-grey-15.ecl-col-12.ecl-col-m-6.ecl-col-l-4');
+    $this->assertCount(2, $speakers_items);
+    $portrait = $this->assertSession()->elementExists('css', '.ecl-u-flex-shrink-0.ecl-u-mr-s.ecl-u-media-a-s.ecl-u-media-bg-size-contain.ecl-u-media-bg-repeat-none', $speakers_items[0]);
+    $this->assertStringContainsString('oe_theme/images/user_icon.svg', $portrait->getAttribute('style'));
+    $meta = $this->assertSession()->elementExists('css', '.ecl-content-item__meta.ecl-u-type-s.ecl-u-type-color-grey-75.ecl-u-mb-xs.ecl-u-type-uppercase', $speakers_items[0]);
+    $this->assertEquals('event role 1', $meta->getText());
+    $link = $this->assertSession()->elementExists('css', '.ecl-link.ecl-link--standalone.ecl-u-type-bold.ecl-u-type-uppercase', $speakers_items[0]);
+    $this->assertStringContainsString($person->toUrl()->toString(), $link->getAttribute('href'));
+    $this->assertEquals($person->label(), $link->getText());
```

Comment:
I noticed that the person name is capitalized.
![image](https://user-images.githubusercontent.com/22004498/143262492-0eb4e5ef-6813-4fc0-8681-20bf6fa2a2e0.png)